tests/foreman/api/test_subnet.py

```diff
@@ -81,8 +133,25 @@ def test_positive_create_with_parameter_and_valid_separator(self):
         :expectedresults: The parameter with multiple names separated by valid
```

Comment:
Does application treat 'test/test2' as two different names that I can refer in future. I meant 'test' and 'test2' specifically?

Reply:
Its dependent on the host which will use that subnet parameter in its environment. but 'test/test2' will go as one value.